templates/email-signatures.md

```diff
@@ -0,0 +1,35 @@
+# Email Signatures
+
+A few examples to base your signature off of. Choose your own title, it can be whatever you wish.
+
+```
+Richard Hendricks
+
+CEO
+---------------
+Niteo
+
+E: rh@niteo.co
+W: www.niteo.co
+```
+
+```
+Erlich Bachmann
+
+Backend Specialist
+------------------
+Niteo
+
+E: eb@niteo.co
+W: www.niteo.co
+```
+```
+Monica Hall
+
+Senior (con)Sultan
+------------------
+Niteo
```

Comment:
I very much prefer having the `.co` suffix. There are a gazillion "Niteo" organizations around the Web, but `niteo.co` can only be us.

Also, I just started using a slimmed down version of the signature, should we include it?

```
Monica Hall, Senior (con)Sultan
niteo.co | +386 40 123 456 | skype: monica.hall | @mhall
```

Reply:
@zupo, For now, we are using just "Niteo" everywhere. If we're changing this to Niteo.co, then we need to go back and change everywhere else. We don't have a guideline for this yet, so keep this as Niteo for now, then let's discuss how to proceed on IRL.

I like your slimmed down version.

Reply:
@dmurko: sorry but that is not true. ATM, in master, the email signature templates have `Niteo.co`. And I would not want to change that to `Niteo` without discussion.